### DISCUSSION TITLE
How to force value on select

### DISCUSSION THREAD

**Discussion by estelleal** [orchidjs/tom-select#317] [Q&A]
Hi,

I am trying to link two tom select together, so when one of them will change, I need to force change on the second one.
I search in the documentation but i didn't found a method to change a selected value in JS.

Someone already got this problem ?

Thanks

**Comment by iamB0rgy** [comment]
> Hi,
> 
> I am trying to link two tom select together, so when one of them will change, I need to force change on the second one. I search in the documentation but i didn't found a method to change a selected value in JS.
> 
> Someone already got this problem ?
> 
> Thanks

I have the same issue. Did you find a solution for this after a year of searching?
